src/test/java/com/google/sps/servlets/PotentialMatchesServletTest.java

```diff
@@ -199,14 +233,35 @@ public void fourUsersOneMutualConnection() throws Exception {
   */
   private String execute(String userIDToFetch) throws IOException{
     when(mockRequest.getParameter(USER_ID_REQUEST_URL_PARAM)).thenReturn(userIDToFetch);
+    
+    responseWriter = new StringWriter();
+    writer = new PrintWriter(responseWriter, true);
+    when(mockResponse.getWriter()).thenReturn(writer);
 
     servletUnderTest.doGet(mockRequest, mockResponse);
     
     String result = responseWriter.toString();
     JSONObject jsonResponse = new JSONObject(result);
+
     return jsonResponse.getString(MATCHINFO_NEXT_MATCH_ID_FIELD);
   }
 
+  /**
+  * Checks if a user's match information was stored in datastore correctly
+  *
+  * @param currUserID The user ID of the user who's match information is getting checked
+  * @param expectedMatches The list of potential matches that are expected to be in datastore
+  */
+  private void datastoreAssertions(String currUserID, List<String> expectedMatches) {
```

Comment:
nit: rename to what the method does, e.g. `assertUserMatchesInDatastore`

Reply:
Got it!